client/features/addToCartButton/AddToCartButton.js

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { selectMe } from '../auth/authSlice';
+
+// Bryan, from your product's components you should be sending a product object and a quantity number (can be 1 for now)
+const AddToCartButton = (product, quantity) => {
+  const data = useSelector(selectMe);
+  const userId = data.id;
+  const productId = product.id;
+
+  const handleAddToCart = async (evt) => {
```

Comment:
may have this wrong but i think button of type submit triggers a re-render - might want the evt.preventDefault() if that is the case

Reply:
You're totally right. Adding this to the handler now.